admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx

```diff
@@ -12,36 +10,66 @@ import {
 import { getAllChats } from "../../redux/chat-list.slice";
 import { useNavigate } from "react-router";
 import { IChatInterface } from "../../../../interfaces/chat.interface";
+import { SettingsSvg } from "../settings";
 
 export const ChatListWidget = () => {
   const { list } = useAppSelector((state) => state.chats);
   const dispatch = useAppDispatch();
   useEffect(() => {
     dispatch(getAllChats());
   }, []);
+  console.log(list);
   const navigate = useNavigate();
   const handleOnClick = useCallback(
     (chat: IChatInterface) => navigate(`/chat/${chat.id}`, { replace: true }),
     [navigate]
   );
+  console.log(list[0]);
   return (
     <>
-      <ChatListWrapper>
-        <Widget name={"Chat list widget"}>
-          <ChatListUL>
-            {list.map((chat) => (
-              <li key={`widget-chat-list--${chat.id}`}>
-                <Chat onClick={() => handleOnClick(chat)}>
-                  <ChatPhotoWrapper>
-                    <ChatPhoto>{chat.title[0].toUpperCase()}</ChatPhoto>
-                  </ChatPhotoWrapper>
+      {/*<ChatListWrapper>*/}
+      {/*  <Widget name={"Chat list widget"}>*/}
+      {/*    <ChatListUL>*/}
+      {/*      {list.map((chat) => (*/}
+      {/*        <li key={`widget-chat-list--${chat.id}`}>*/}
+      {/*          <Chat onClick={() => handleOnClick(chat)}>*/}
+      {/*            <ChatPhotoWrapper>*/}
+      {/*              <ChatPhoto>{chat.title[0].toUpperCase()}</ChatPhoto>*/}
+      {/*            </ChatPhotoWrapper>*/}
+      {/*            <ChatTitle>{chat.title}</ChatTitle>*/}
+      {/*          </Chat>*/}
+      {/*        </li>*/}
+      {/*      ))}*/}
+      {/*    </ChatListUL>*/}
+      {/*  </Widget>*/}
+      {/*</ChatListWrapper>*/}
+      {list.map((chat) => (
+        <ChatListWrapper
+          key={`widget-chat-list--${chat.id}`}
+          style={{ minWidth: 280, width: "100%" }}
+        >
+          <Widget name={""} onClick={() => handleOnClick(chat)}>
+            <div style={{ display: "flex", justifyContent: "space-between" }}>
+              <div
+                style={{
+                  display: "flex",
+                  alignItems: "center",
+                  gap: 20,
+                }}
+              >
+                <ChatPhotoWrapper>
+                  <ChatPhoto>{chat.title[0].toUpperCase()}</ChatPhoto>
+                </ChatPhotoWrapper>
+                <div>
```

Comment:
better use style components

Reply:
Why is it better to use style components here if we don't need any styles?

Reply:
Okay, I agree with you.